sdk/identity/identity/review/identity.api.md

```diff
@@ -37,6 +37,13 @@ export class AuthorizationCodeCredential implements TokenCredential {
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
     }
 
+// @public
+export class AzureCliCredential implements TokenCredential {
+    constructor();
+    protected getAzureCliAccessToken(resource: string): Promise<unknown>;
```

Comment:
Should this be refactored so it's not exposed?

Reply:
Is there not another way to mock / test this? I'm not a big fan of adding public surface area for testing purposes. If we can't fix it for this preview could we file an issue to fix before GA?

Reply:
This is coming from the mocking that is used for testing, since we can't override that method if it's private.

Can look into a better approach for the next preview.